packages/react/src/recipes/seo/helpers.ts

```diff
@@ -0,0 +1,70 @@
+import {ColorModesEnum} from '../../ThemeProvider'
+
+export const themes = ['ai', 'collaboration', 'enterprise', 'security', 'productivity'] as const
+
+export type Themes = (typeof themes)[number]
+
+type Palette = {
+  accent1: string
+  accent2: string
+}
+
+export type ThemeDetailsMap = {
+  [theme in Themes]: {
+    [ColorModesEnum.LIGHT]: Palette
+    [ColorModesEnum.DARK]: Palette
+  }
+}
+
+export const themeDetailsMap: ThemeDetailsMap = {
+  ai: {
+    light: {
+      accent1: 'var(--base-color-scale-blue-5)',
+      accent2: 'var(--base-color-scale-lime-1)',
```

Comment:
@rezrah these are technically a stop-gap solution (see full --base-color / brand color matching [here](https://www.figma.com/design/n7dHtVVkFOvDLyxWaNfKou/CTA-banner-expressive-variant?node-id=1-13765&t=MljT3KnlwWpflZki-1) ). I'd recommend starting here and since Brand is working with Gretel (agency) to rework our color strategy, that may indeed change. That's why I'd recommend sticking to our color scale for the time being